### PR TITLE
Add August Meeting

### PR DESCRIPTION
**Pull request by glenjarvis**
Ticket: None

Add August Meeting

### CODE REVIEW

File: content/meetings/Meeting-2019-08-22.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* Python manipulation of Google Drive permissions:
+* * Make file world-readable
+* * Grant write-privilege to a gmail-address
+* List Sharable-URLs of files in Google Drive
+* Google Sheets + Python
+* Create a Sheet
+* Understand "A1 Notation"
+* Add rows to a Sheet
+* * Make Sheet world-readable
+* * Grant write-privilege to a gmail-address
```
Comment (glenjarvis) [inline content/meetings/Meeting-2019-08-22.rst:85]: @bdbaddog  Is this the proper way to have a nested bullet?

Comment (bdbaddog) [reply content/meetings/Meeting-2019-08-22.rst:85]: I think it's just indented and start with dashes.. I'll merge and then get it right.

Comment (glenjarvis) [reply content/meetings/Meeting-2019-08-22.rst:85]: @bdbaddog Okay, I'll watch what your next commit so that I know for next time.

Comment (bdbaddog) [reply content/meetings/Meeting-2019-08-22.rst:85]: Like this:

```
* File upload to Google Drive
* Python manipulation of Google Drive permissions

  * Make file world-readable
  * Grant write-privilege to a gmail-address

* List Sharable-URLs of files in Google Drive
* Google Sheets + Python
* Create a Sheet
* Understand "A1 Notation"
* Add rows to a Sheet

  * Make Sheet world-readable
  * Grant write-privilege to a gmail-address

* Sheets are complicated, So is the API.
```

Comment (glenjarvis) [reply content/meetings/Meeting-2019-08-22.rst:85]: 👍

**Pull request merged**